charts/jaeger/templates/collector-deploy.yaml

```diff
@@ -66,6 +66,7 @@ spec:
           - name: {{ $key | quote }}
             value: {{ $value | quote }}
           {{- end }}
+          {{- toYaml .Values.storage.kafka.extraEnv | nindent 10 }}
```

Comment:
I think we need the if check here `{{- if .Values.storage.kafka.extraEnv }}`

Reply:
good catch, thanks!